README.md

```diff
@@ -2,4 +2,329 @@
 
 ## Objectives
 
-## Introduction
\ No newline at end of file
+1. Understand Swift collection types as generic collections.
+2. Recognize some implications of the pass-by-value nature of Swift collections.
+3. Create an explicitly-typed array.
+4. Use subscripting to interact with an array.
+5. Use a few array methods and properties to interact with an array.
+6. Create an explicitly-typed dictionary.
+7. Use subscripting to read a dictionary.
+8. Use subscripting to overwrite or add values for a key in a dictionary.
+9. Use the key-value Tuple returned by iterating over a dictionary with a `for-in` loop.
```

Comment:
yeah, it's really more material than belongs in a single reading, and what's here is VERY cursory